tests/cypress/support/widgets/PythonTransformationDialogWidget.ts

```diff
@@ -38,18 +38,16 @@ export default class PythonTransformationDialogWidget extends Widget {
         browser.setInputValue(selectors.codeInput(id), code);
     }
 
-    getCode(id = 0) {
-        return browser.execute((win: AUTWindow) => {
-            const { document } = win;
-            const element = document.getElementById(selectors.pythonOutput(id, true));
-            return element
-                ?.getElementsByClassName("cm-content")[0]
-                .cmView.view.state.doc.toString();
-        });
-    }
-
-    getPythonOutput(id = 0) {
-        return browser.getInputValue(selectors.pythonOutput(id));
+    getCode(id = 0): Cypress.Chainable<string> {
+        const elementSelector = selectors.pythonOutput(id, true);
+        return cy
+            .get(`#${elementSelector} .cm-content`, { timeout: 20000 })
```

Comment:
Let's not use magic numbers, please. Let's define this as a variable and import from settings - there's probably already long, medium, short wait or smth like that

Reply:
It was in tests_legacy. tests didn't import or inherit anything from them